tracker/src/androidTest/java/com/google/android/apps/santatracker/tracker/viewmodel/TrackerViewModelTest.kt

```diff
@@ -97,247 +92,9 @@ class TrackerViewModelTest {
-                    LiveDataTestUtil.getValue(viewModel.santaState)
-            assertThat(santaState.isTraveling, `is`(false))
-            assertThat(santaState.destination, `is`(sydney))
-            val visited = santaState.visitedDestinations
-            assertThat(visited.size, `is`(BRISBANE_INDEX + 1))
-            assertThat(visited[visited.size - 1], `is`(brisbane))
-        })
-
-        `when`(mockClock.nowMillis()).thenReturn(SYDNEY_DEPARTURE +
-                TimeUnit.SECONDS.toMillis(10))
-
-        viewModel.updateSantaState(destinationList, streamEntryList)
-        InstrumentationRegistry.getInstrumentation().waitForIdleSync()
-
-        activityRule.runOnUiThread({
-            assertThat(viewModel.destinationIndex, `is`(WOLLONGONG_INDEX))
-            // Asserting this in the main thread because LiveData can be observed only from
-            // the main thread.
-            val santaState: TrackerViewModel.SantaState =
-                    LiveDataTestUtil.getValue(viewModel.santaState)
-            assertThat(santaState.isTraveling, `is`(true))
-            assertThat(santaState.destination, `is`(wollongong))
-            val visited = santaState.visitedDestinations
-            assertThat(visited.size, `is`(SYDNEY_INDEX + 1))
-            assertThat(visited[visited.size - 1], `is`(sydney))
-        })
-    }
-
-    @Test
-    @MediumTest
-    fun `testUpdateSantaState_between_last_arrival`() {
-        `when`(mockClock.nowMillis()).thenReturn(LAST_ARRIVAL -
-                TimeUnit.SECONDS.toMillis(10))
-        val lastDestination = destinationList[destinationList.size - 1]
-        activityRule.runOnUiThread({
-            viewModel.initializeIndices(mockClock.nowMillis(), destinationList, streamEntryList)
-            viewModel.initializeSantaState(destinationList, streamEntryList)
-            assertThat(viewModel.destinationIndex, `is`(destinationList.size - 1))
-        })
-        InstrumentationRegistry.getInstrumentation().waitForIdleSync()
-        activityRule.runOnUiThread({
-            val santaState: TrackerViewModel.SantaState =
-                    LiveDataTestUtil.getValue(viewModel.santaState)
-            assertThat(santaState.isTraveling, `is`(true))
-            assertThat(santaState.destination, `is`(lastDestination))
-            val visited = santaState.visitedDestinations
-            assertThat(visited.size, `is`(destinationList.size - 1))
-            assertThat(LiveDataTestUtil.getValue(viewModel.finishedTraveling), `is`(false))
-        })
-
-        `when`(mockClock.nowMillis()).thenReturn(LAST_ARRIVAL +
-                TimeUnit.SECONDS.toMillis(10))
-
-        viewModel.updateSantaState(destinationList, streamEntryList)
-        InstrumentationRegistry.getInstrumentation().waitForIdleSync()
-
-        activityRule.runOnUiThread({
-            assertThat(LiveDataTestUtil.getValue(viewModel.finishedTraveling), `is`(true))
-        })
```

Comment:
Why remove these tests?

Reply:
That is a question for upstream. @gavra0 Any ideas?

Reply:
I think we started from a different SHA, which probably did not contain these tests. Anyhow, we only measure `assembleDebug`, so test sources do not really matter.